sycl/doc/extensions/Assert/SYCL_ONEAPI_ASSERT.asciidoc

```diff
@@ -0,0 +1,144 @@
+= SYCL_EXT_ONEAPI_ASSERT
+
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+:blank: pass:[ +]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+// This is necessary for asciidoc, but not for asciidoctor
+:cpp: C++
+
+== Notice
+IMPORTANT: This specification is a draft.
+
+Copyright (c) 2021-2021 Intel Corporation. All rights reserved.
+
+NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are
+trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc.
+used by permission by Khronos.
+
+NOTE: This document is better viewed when rendered as html with asciidoctor.
+GitHub does not render image icons.
+
+== Dependencies
+
+This extension is written against the SYCL 2020 specification, Revision 3.
+
+== Status
+
+Working Draft
+
+This is a preview extension specification, intended to provide early access to
+a feature for review and community feedback. When the feature matures, this
+specification may be released as a formal extension.
+
+Because the interfaces defined by this specification are not final and are
+subject to change they are not intended to be used by shipping software
+products.
+
+== Introduction
+This extension adds the ability for device code to call the C++ `assert()`
+macro. The behavior of `assert()` in device code is similar to its behavior in
+host code. If the asserted condition is false, a message is printed to `stderr`
+and then the program aborts with `std::abort()`.
+
+The format of the assert message is unspecified, but it will always include the
+text of the failing expression, the values of the standard macros `__FILE__` and
+`__LINE__`, and the value of the standard variable `__func__`. If the failing
```

Comment:
```suggestion
text of the failing expression, the values of the standard macros `+__FILE__+` and
`+__LINE__+`, and the value of the standard variable `+__func__+`. If the failing
```

The underbars aren't rendering in the HTML output.  I think the suggestion above will fix it, but please check.

Reply:
Fixed.